src/repos/PublicationRepo.ts

```diff
@@ -13,16 +17,19 @@ const addPublicationsToDB = async (): Promise<void> => {
 
   const publications = [];
   for (const publication of publicationsDB) {
-    const { bio, rssURL, imageURL, name, websiteURL, rssName } = publication;
+    const { bio, bioShort, rssName, rssURL, name, slug, websiteURL } = publication;
     publications.push(
       Object.assign(new Publication(), {
         _id: new ObjectId().toString(),
+        backgroundImageURL: getImageURL(slug, 'background'),
         bio,
-        rssURL,
-        imageURL,
+        bioShort,
         name,
-        websiteURL,
+        profileImageURL: getImageURL(slug, 'profile'),
```

Comment:
not sure if I like this more or less than
```
const { bio, bioShort, rssName, rssURL, name, slug, websiteURL } = publication;
const { backgroundImageURL, profileImageURL } = getImageURLs(slug) 
publications.push(
    Object.assign(new Publication(), {
        ...
        backgroundImageURL,
        ...
        profileImageURL,
        ...
```
thoughts on the two styles?
(this is not an important issue, I was just curious to think of another way of going about it)

Reply:
Yeah, it looks a lot better moving them out. I used a Typescript tuple here though because otherwise the function signature with types becomes a little long.

Reply:
wouldn't using a tuple here constrict us to only return a profile and a background image? can't imagine a situation where it occurs rn but we might potentially add other publication assets in the future